__tests__/app.test.js

```diff
@@ -188,4 +188,87 @@ describe("/api/articles/:article_id/comments", ()=> {
             expect(body.msg).toBe("Not found")
         })
     })
+
+    test("POST 201: Responds with the comment posted", ()=> {
+        return request(app)
+        .post("/api/articles/7/comments")
+        .send({
+            username: "lurker",
+            body:"This is just something I want to test, Thank you!"
+        })
+        .expect(201)
+        .then(({body}) => {
+            const {comment} = body;
+            expect(comment).toEqual({
+                author: "lurker",
+                body:"This is just something I want to test, Thank you!"
+            })
+        })
```

Comment:
^^ Is this everything you would expect to be on a comment that exists in the database? Have a think about what else would be on the object - i.e. created_at etc :)